### ISSUE TITLE
Don't get any results for the newly created database

### ISSUE THREAD

**Issue by NaomiN** [MicrosoftDocs/sql-docs#5098]
I tried different queries for the database I just created using the documented views, but they returned no results even though I can see that, say, Production.Products table depends on the Production.Categories table.

BTW, there is still the same problem - when I try to add a new comment under the article (in that particular case I was reading about sql_expression_dependencies view), the new tab opens to add comment, but the original article is automatically closed, so when I try to add my issue it's not even referenced and I cannot see it anymore. I'm using Google Chrome.

Here is the query I tried

SELECT OBJECT_SCHEMA_NAME ( referencing_id ) AS referencing_schema_name,  
    OBJECT_NAME(referencing_id) AS referencing_entity_name,   
    o.type_desc AS referencing_desciption,   
    COALESCE(COL_NAME(referencing_id, referencing_minor_id), '(n/a)') AS referencing_minor_id,   
    referencing_class_desc, referenced_class_desc,  
    referenced_server_name, referenced_database_name, referenced_schema_name,  
    referenced_entity_name,   
    COALESCE(COL_NAME(referenced_id, referenced_minor_id), '(n/a)') AS referenced_column_name,  
    is_caller_dependent, is_ambiguous  
FROM sys.sql_expression_dependencies AS sed  
INNER JOIN sys.objects AS o ON sed.referencing_id = o.object_id  
WHERE referenced_id = OBJECT_ID(N'Production.Categories);  
GO  

I'm using database from the script attached to this article
https://www.itprotoday.com/sql-server/logical-query-processing-what-it-and-what-it-means-you

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6ada44a9-e5a1-65b3-3bb2-9ebac215e5ab
* Version Independent ID: 470c3c15-d19d-e29f-c5f9-0824881deaac
* Content: [sys.sql_expression_dependencies (Transact-SQL) - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/system-catalog-views/sys-sql-expression-dependencies-transact-sql?view=sql-server-ver15)
* Content Source: [docs/relational-databases/system-catalog-views/sys-sql-expression-dependencies-transact-sql.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/system-catalog-views/sys-sql-expression-dependencies-transact-sql.md)
* Product: **sql**
* Technology: **system-objects**
* GitHub Login: @CarlRabeler
* Microsoft Alias: **carlrab**

**Comment by CarlRabeler** [comment]
Run the following query and let me know what results are returned: SELECT * FROM Production.Categories

**Comment by NaomiN** [comment]
Eight rows returned:

categoryid	categoryname	description
1	Beverages	Soft drinks, coffees, teas, beers, and ales
2	Condiments	Sweet and savory sauces, relishes, spreads, and seasonings
3	Confections	Desserts, candies, and sweet breads
4	Dairy Products	Cheeses
5	Grains/Cereals	Breads, crackers, pasta, and cereal
6	Meat/Poultry	Prepared meats
7	Produce	Dried fruit and bean curd
8	Seafood	Seaweed and fish

**Comment by NaomiN** [comment]
More details (if needed) in this thread https://social.msdn.microsoft.com/Forums/sqlserver/en-US/319e5963-d28e-4b53-9fec-e2fe78259f33/how-to-rename-a-table-and-column-using-a-script-to-upgrade-database?forum=transactsql

**Comment by CarlRabeler** [comment]
I do not understand your question. 
   1. When I run the examples in the referenced article against the AdventureWorks database - they all return the expected information. 
   2. I do not see a table called Production.Categories in the Adventure Works database.

So, I do not understand what the documentation issue is with the referenced article.

Thank you.
Carl

**Comment by NaomiN** [comment]
Hi Carl,

I was thinking (incorrectly) that the sys.dm_referencing_entities or sys.dm_referenced_entities views are supposed to give me dependencies such as Parent/Child tables. However, Erland Sommarskog already set me straight in the thread I referenced. The documentation was not very clear and I believe I originally got these views from another article I was reading (using sp_depends as a search term in Google). Anyway, I think I can probably close this question (and as I said originally, I was not using AdventureWorks, but rather a database I downloaded from Itzik Ben-Gan article about Logical processing of the query).
